### PR TITLE
Fixed some UI problem and upgrade to latest Bulma

### PR DESCRIPTION
**Pull request by michioxd**
- Upgraded to [Bulma 1.0.1](https://github.com/jgthms/bulma/blob/main/CHANGELOG.md#101)
- Now have dark mode (Bulma built-in auto detect color scheme)

  image

### CODE REVIEW

File: _includes/nav.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
-<nav class="navbar is-info is-fixed-top" role="navigation" aria-label="main navigation">
+<nav class="navbar is-fixed-top is-link" role="navigation" aria-label="main navigation">
   <div class="container" id="container">
     <div class="navbar-brand">
-        <img class="navlogo" style="width:50px;height:52px;" src="/img/favicon.png">
```
Comment (Smu1zel) [inline _includes/nav.html:4]: Was there a reason for this? It causes the bug found at #9 and also affects Safari 12.

Comment (michioxd) [reply _includes/nav.html:4]: Oh I got it

image

Comment (michioxd) [reply _includes/nav.html:4]: @Smu1zel Maybe can you open that PR again? :)

Comment (Smu1zel) [reply _includes/nav.html:4]: > @Smu1zel Maybe can you open that PR again? :)

Sure

Comment (Smu1zel) [reply _includes/nav.html:4]: It's now merged. 

Regardless, thank you for the PR.

**Pull request merged**